### PR TITLE
PHHS Basalt to EQR120 SM Recorder Swap

### PR DESCRIPTION
**Pull request by Thomas-Benson**
- Removing Basalt (1277) from PHHS LC 20
- Replacing with EQR120 (46661) at same location
- Using latest EQR120 firmware version 4.3.2
- Updated streams to continuous from 2020-02-09
- Streams affected: PHHS LC 20 HN-12Z

### CODE REVIEW

File: install/firmware.csv
Comment (mnaguit) [inline install/firmware.csv:1]: use opening time of recorder? thanks @Thomas-Benson

**Review by mnaguit — CHANGES_REQUESTED** [review]
tweak opening time of firmware, to match opening time of recorder

File: install/firmware.csv
Comment (Thomas-Benson) [reply install/firmware.csv:1]: Fixed, my bad for using the closing time instead.

Comment (mnaguit) [reply install/firmware.csv:1]: All good @Thomas-Benson 😄

**Review by mnaguit — APPROVED** [review]

**Review by gva6500 — APPROVED** [review]

**Pull request merged**